src/client/post/EditPost/EditPost.js

```diff
@@ -224,6 +229,18 @@ class EditPost extends Component {
       draftId,
       ...settings,
     };
+    const isSubmitPost =  isSubmit && !isEmpty(defaultObjects);
+    const postObjects = isSubmitPost ? [...defaultObjects, ...linkedObjects] : linkedObjects;
+
+    if (isSubmitPost) {
+      defaultObjects.forEach(obj => {
+          const objName = obj.name || obj.default_name;
+          postData.body += `\n[${objName}](${getObjectUrl(
+            obj.id || obj.author_permlink,
+          )})&nbsp;\n`;
+        },
+      );
```

Comment:
move this logic to handleSubmit function

Reply:
++